src/frontend/src/app/scheduled-process/scheduled-process.component.ts

```diff
@@ -33,6 +33,7 @@ export class ScheduledProcessComponent implements OnInit {
   public invalidRepeat: boolean = false
   public invalidTime: boolean = false
   public selectedTaskType = 'Latest' // Default is LATEST  
+  public scheduler_name: string; 
```

Comment:
this var introduced but never used? or it was used and we found declaration error while doing error simulation/ handling?

Reply:
This is used in html directly in `[(ngModel)]="scheduler_name"` for two way binding.